src/DeterministicVI.jl

```diff
@@ -16,16 +16,23 @@ export ElboArgs
 
 """
 ElboArgs stores the arguments needed to evaluate the variational objective
-function
+function.
 """
 type ElboArgs{NumType <: Number}
     S::Int64
     N::Int64
+
+    # The number of components in the point spread function.
+    psf_K::Int64
     images::Vector{TiledImage}
     vp::VariationalParams{NumType}
     tile_source_map::Vector{Matrix{Vector{Int}}}
     patches::Matrix{SkyPatch}
     active_sources::Vector{Int}
+
+    # The number of standard deviations away below which a bivariate normal
+    # will be calculated.  Set to Inf to evaluate bivariate normals at all points.
+    num_allowed_sd::Float64
```

Comment:
> Why would we voluntarily introduce discontinuities?
Because it saves a considerable amount of computation.

> What's wrong with fixing the active pixels a priori?
We do fix the active pixels a priori.  Maybe you mean to have a set of active pixels per source?  That's a good idea, but a more intrusive change.  Even with either of these plans, this is not a no-op, since a single multivariate Gaussian may not contribute meaningfully to a large number of pixels.

Re: the number of standard deviations and different optimizers, these questions can only be answered by experience.  That is why you can set it in ElboArgs and it is Inf by default.  Suffice to say unit tests pass (and pass faster) when I set this to four.